tests/stub/transport.py

```diff
@@ -38,13 +38,9 @@ def setUp(self):
         self._session = self._driver.session("w")
 
     def tearDown(self):
-<<<<<<< HEAD
         self._backend.close()
```

Comment:
Oops, how did that merge conflict end up here :fearful: 
`self._backend.close()` can also go, as it's part of `TestkitTestCase.tearDown`.

Reply:
Sure, `self._backend.close()` removed.